data-structures-queues/get-front.js

```diff
@@ -0,0 +1,4 @@
+export default function getFront(queue) {
+  if (queue.peek() === undefined) return undefined;
```

Comment:
If you think about it a moment you will realize that this line of code is not necessary.